packages/protocol/test/governance/epochrewards.ts

```diff
@@ -26,6 +28,7 @@ const MockElection: MockElectionContract = artifacts.require('MockElection')
 const MockGoldToken: MockGoldTokenContract = artifacts.require('MockGoldToken')
 const MockSortedOracles: MockSortedOraclesContract = artifacts.require('MockSortedOracles')
 const Registry: RegistryContract = artifacts.require('Registry')
+const Reserve: ReserveContract = artifacts.require('Reserve')
```

Comment:
Can we use a `MockReserve` here instead?

Reply:
We can use MockReserve.  I thought it might be desirable to use the actual ReserveContract until circumstances demanded MockReserve?

Reply:
I don't feel strongly one way or the other